test/lib/linter/xmlTemplate/snapshots/transpiler.ts.md

```diff
@@ -26,21 +26,21 @@ Generated by [AVA](https://avajs.dev).
         text: "Hello \\"World\\"",␊
     });␊
     ␊
-    export default const oView = new View({␊
+    const oView = new View({␊
```

Comment:
why not `export new View({`?

Reply:
Basically, this is the simplest solution that will not break anything.
The impact of this change is just the export of XMLView. 
However, if we want to implement `export default new View...`, then we'll need to also modify the code for variable declaration & initialization which resides in another module and will definitelly imact the other scenarios